modules/aggregations/src/yamlRestTest/resources/rest-api-spec/test/aggregations/percentiles_tdigest_metric.yml

```diff
@@ -187,6 +187,65 @@ setup:
   - close_to:  { aggregations.percentiles_double.values.95\.0: { value: 146.0, error: 5.0 } }
   - close_to:  { aggregations.percentiles_double.values.99\.0: { value: 150.0, error: 1.0 } }
 
+
+---
+"Execution hint":
+  - skip:
+      features: close_to
+      version: " - 8.8.99"
+      reason: t-digest forked in 8.9.0
+  - do:
+      search:
+        body:
+          aggs:
+            percentiles_int:
+              percentiles:
+                field: int_field
+                tdigest:
+                  execution_hint: high_accuracy
+
+  - match: { hits.total.value: 4 }
+  - length: { hits.hits: 4 }
+  - close_to:  { aggregations.percentiles_int.values.1\.0: { value: 5.0, error: 5.0 } }
```

Comment:
How is the `error` value calculated? I am asking because if we do value +- error for some percentiles here the actual ranges overlap. For instance the 1 and 5 percentiles...maybe this is just a result of the algorithm error.

Reply:
There's a big difference here between AVLTreeDigest and SortingDigest, I think it was 1.0 vs 10.0, so I used a mid value to cover both. 

In general, each version produces values that are reasonably spread, but the ranges to cover both versions overlap indeed. I wish there was an option to provide alternative set of results, with `match` or `close_to`.